buildbot-buildmaster/buildbot/master.cfg

```diff
@@ -0,0 +1,276 @@
+factoryWin32.addStep(steps.ShellCommand(description=desc, descriptionDone=descDone, command=[windowsShell, testdir + "buildAndTest.sh", "win32", "8"]))
+factoryWin32.addStep(steps.ShellCommand(description=desc, descriptionDone=descDone, command=[windowsShell, testdir + "buildAndTest.sh", "win32", "16"]))
+factoryWin32.addStep(steps.ShellCommand(description=desc, descriptionDone=descDone, command=[windowsShell, testdir + "buildAndTest.sh", "win32", "32"]))
+
+####### BUILDERS
+
+# The 'builders' list defines the Builders, which tell Buildbot how to perform a build:
+# what steps, and which workers can execute them.  Note that any particular build will
+# only take place on one worker.
+
+c['builders'] = []
+
+c['builders'].append(
+    util.BuilderConfig(name="x86_64",
+      workernames=[multiarch_worker],
+      factory=factoryLinux_x86_64))
+
+c['builders'].append(
+    util.BuilderConfig(name="i386",
+      workernames=[multiarch_worker],
+      factory=factoryLinux_i386))
+
+c['builders'].append(
+    util.BuilderConfig(name="arm",
+      workernames=[multiarch_worker],
+      factory=factoryLinux_armhf))
+
+c['builders'].append(
+    util.BuilderConfig(name="armv6_native",
+      workernames=[armv6_worker],
+      factory=factory_arm))
+
+c['builders'].append(
+    util.BuilderConfig(name="armv7_native",
+      workernames=[armv7_worker],
+      factory=factory_arm))
+
+c['builders'].append(
+    util.BuilderConfig(name="armv8_native",
+      workernames=[armv8_worker],
+      factory=factory_arm))
+
+c['builders'].append(
+    util.BuilderConfig(name="arm64v8_native",
+      workernames=[arm64v8_worker],
+      factory=factory_aarch64))
+
+c['builders'].append(
+    util.BuilderConfig(name="win_x64",
+      workernames=[windows_worker],
+      factory=factoryWin_x64))
+
+c['builders'].append(
+    util.BuilderConfig(name="win32",
+      workernames=[windows_worker],
+      factory=factoryWin32))
+
+################
+
+gs = reporters.GitHubStatusPush(token=util.Secret("githubToken"),
+                                context=util.Interpolate("buildbot/%(prop:buildername)s"),
+                                startDescription='Build started.',
+                                endDescription='Build end',
+                                verbose=True)
+
+c['services'] = []
+#c['services'].append(gs)
+
+####### STATUS TARGETS
+
+# 'status' is a list of Status Targets. The results of each build will be
+# pushed to these targets. buildbot/status/*.py has a variety to choose from,
+# like IRC bots.
+
+#c['status'] = []
+
+####### PROJECT IDENTITY
+
+# the 'title' string will appear at the top of this buildbot installation's
+# home pages (linked to the 'titleURL').
+
+c['title'] = os.environ.get("BUILDBOT_TITLE", "buildbot")
+c['titleURL'] = os.environ.get("BUILDBOT_TITLE_URL", "https://docs.buildbot.net/latest/developer/config.html#")
```

Comment:
Is this link desirable?

Reply:
Changed to the data-compressor repository. This URL is used as a fallback if BUILDBOT_TITLE_URL is not set.